components/List/List.tsx

```diff
@@ -15,7 +15,7 @@ const List: React.FunctionComponent<ListProps> = ({ list, i18n }) => {
 
   const followArtist = (artistData: ListEl) => {
     setArtistLoading(artistData.name);
-    fetch('/me/unfollow', {
+    fetch('https://release-racconBE.com/todos/me/unfollow', {
```

Comment:
use real url

Reply:
Could we not make the base url configurable and refer to it here instead? Same goes for the below place holder comments